scripts/release_info.py

```diff
@@ -33,8 +33,7 @@
 # pylint: enable=wrong-import-position
 
 GIT_CMD_GET_STATUS = 'git status'
-GIT_CMD_GET_TAGS = 'git tag'
-GIT_CMD_GET_LCA_WITH_DEVELOP = 'git merge-base develop %s'
+GIT_CMD_GET_NEW_COMMITS = 'git cherry %s -v'
```

Comment:
GIT_CMD_TEMPLATE_GET_NEW_COMMITS

(to clarify that it still needs interpolation)